OKR.md

```diff
@@ -33,4 +34,5 @@ Quarterly statement of OKRs (Objectives and Key Results) for the "IPFS in Web Br
 **Structure documentation and discussion around the primary Browser vendor concerns:**
 
 - `dweb:` proposal is documented
+- `ipfs://` (short term URL-based solution) is documented
```

Comment:
Good stuff @lidel. 

> short term URL-based solution
Avoid `short term`. I think the idea is that all the proposed address forms are legitimate, and will be supported.

Reply:
good point, reorganized in 35dc0ce